app/src/main/main.js

```diff
@@ -344,3 +345,12 @@ ipcMain.on('ask-user-to-save-file', (event, data) => {
     });
   });
 });
+
+function changeOutputDestination() {
```

Comment:
I think it's better to move it to `menus.js`, since it'll be used there (for now) 😄